Backend/router/v1/report.ctrl.js

```diff
@@ -1,41 +1,70 @@
 const models = require('../../models');
 
 const createReport = (req, res) => {
-  models.report.create({
-    what: req.body.data[0].what,
-    location: req.body.data[0].location,
-    time: req.body.data[0].time,
-    who: req.body.data[0].who,
-    details: req.body.data[0].details,
-  }).then((result) => {
-    res.status(201);
-    res.send("good");
-  });
-}
+  if(req.isAuthenticated()){
+    console.log("User id : ", req.user[0].dataValues.id);
+    models.report.create({
+      what: req.body.data[0].what,
+      location: req.body.data[0].location,
+      time: req.body.data[0].time,
+      who: req.body.data[0].who,
+      details: req.body.data[0].details,
+      userID: req.user[0].dataValues.id
+    }).then((result) => {
+      res.status(201);
+      res.json({
+        msg: "make report successfully"
```

Comment:
Not sure if I understand this message - are you suggesting the frontend to make a successful report? Or did you mean 'report made successfully'?

Reply:
the second one. 'report made successfully' and give the result to client.

Reply:
Then please update. When I ask you a question, I'm not expecting you to add another comment explaining. What I want is for you to update the code so I don't have to ask the question at the first place.